### PR TITLE
5.0 embedded holdings

### PR DESCRIPTION
**Pull request by HirschSt**

### CODE REVIEW

File: app/models/institution.rb [hunk 1]
```diff
@@ -78,6 +78,7 @@ class Institution < ApplicationRecord
   attr_accessor :suppress_update_workgroups_trigger
 
   alias_attribute :id_for_fulltext, :id
+  alias_method :referring_holdings, :holdings
```
[the remark this answers is not in the document]
Comment (HirschSt) [reply app/models/institution.rb]: Yes, it's an alias for the ActiveRecord relationship. Your concern is quite ok, I can remove the alias in a few moments...

**Comment by HirschSt** [comment]
Great and Thanks!
I'd like to delete the branches 5.0-embedded_holdings and 5.0-validation
now, ok?

Am Mo., 29. Okt. 2018 um 15:10 Uhr schrieb xhero <notifications@github.com>:

**Comment by xhero** [comment]
ok for me!

**Comment by xhero** [comment]
also, -test is already updated and reindexing

**Comment by HirschSt** [comment]
Probably a bit toooooo fast for me .... 😉

Am Mo., 29. Okt. 2018 um 15:16 Uhr schrieb xhero <notifications@github.com>:

**Pull request merged**